app/enterprise/1.3-x/brain-immunity/install-configure.md

```diff
@@ -14,129 +13,7 @@ Kong Brain and Kong Immunity are installed as add-ons on Kong Enterprise, using
-```
-
-2. Start celery-worker
-```bash
-$ docker run -d --name celery-worker \
-  --network=kong-ee-net \
-  -e "CELERY_BROKER_URL=redis://redis:6379/0" \
-  -e "SQLALCHEMY_DATABASE_URI=postgres://collector:collector@collector-database:5432/collector" \
-  -e "KONG_HOST=kong-ee" \
-  -e "KONG_PORT=8001" \
-  kong-bi \
-  celery worker -l info -A collector.scheduler.celery --concurrency=1
-```
-
-### Step 8. Validate the Collector App Installation
-To complete the Collector App installation, validate the Collector App is working:
-```bash
-curl -i -X GET --url http://localhost:5000/status
-```
-You should receive an HTTP/1.1 200 OK message.
-
-
-### Step 9. Enable and Configure the Collector Plugin using Kong Manager
-
-Using Kong Manager, enable the Collector Plugin. To enable the plugin:
-1. Navigate to the **Workspaces** page.
-2. Click the workspace you want to use. For example, the **default** workspace.
-3. Click **Plugins** in the API Gateway section of the left navigation bar.
-4. Click **New Plugin** which opens a page of plugin options.
-5. Scroll to the Analytics and Monitoring section, and click **Enable** on the **Collector** tile.
-6. The **Create new collector plugin** dialog displays. To **minimally** configure the Collector Plugin:
-** In the **Config.Http Endpoint** field, enter the Collector App endpoint that Kong Enterprise can communicate with. For example, http://collector:5000.
-** The default values populating the remaining fields are valid for a minimal configuration.
-7. Click **Create**. The Collector Plugin is configured.
-
-Using a cURL command enter:
-```bash
-$ curl -X POST localhost:8001/default/plugins \
-    -d name=collector \
-    -d config.http_endpoint=http://collector:5000
-```    
-
-### Step 10. Confirm the Collector Plugin is Enabled and Configured
-1. Click **Alerts** in the left navigation pane, and the Alerts page should say **Collector is connected**.
-2. Click **Service Map** in the left navigation pane, and the Service Map should display with the **Add Service** button.
-
-Using a cURL command, enter:
-```bash
-$ curl localhost:8001/default/collector/status
-```
-
-## Summary
-Collector App and Kong are up and ready to analyze incoming traffic for [alerts](/enterprise/{{page.kong_version}}/brain-immunity/alerts), [auto-generate specs](/enterprise/{{page.kong_version}}/brain-immunity/auto-generated-specs), and display your traffic visually in [Service-Map](/enterprise/{{page.kong_version}}/brain-immunity/service-map).
+## Version Compatibility
 
-For any problems encountered while setting up Collector App, Collector Plugin, or configuring other aspects of Brain and Immunity, please check out [troubleshooting](/enterprise/{{page.kong_version}}/brain-immunity/troubleshooting) for help debugging common problems.
+Kong Brain and Kong Immunity are not compatible with Kong Enterprise v1.3.x. To use Kong Brain and Immunity, you need to upgrade to Kong Enterprise v1.5.x or above.
```

Comment:
There's a bunch of docs for Brain and Immunity though, which shows that at one point, 1.3-x was compatible. I wonder if this section also needs to say that Brain and Immunity is deprecated and not available for download anymore. Something like:

```suggestion
Kong Brain is deprecated, and Kong Immunity is no longer compatible with Kong Enterprise v1.3-x. To use Kong Immunity, you need to upgrade to Kong Enterprise v1.5.x or above.
```